test/e2e/utils.go

```diff
@@ -289,6 +317,12 @@ func WaitAndPollForHTTPResponse(targetURL string, condition ValidateHTTPResponse
 
 func handleSuiteTearDown() {
 	logrus.Info("Entering TearDownSuite()")
+	if saveLogs && !usingOLM {
```

Comment:
Why only when not using OLM?

Reply:
When using OLM we don't install an operator per test suite, there will just be one operator in openshift-operators.  I will update the CI job to collect that.